client/src/components/Tickets/TicketCreateForm.js

```diff
@@ -222,21 +226,40 @@ function TicketCreateForm({
                     Ticket creating error: {creatingErrorMessage}
                   </Alert>
                 )}
-                <Button
-                  type="submit"
-                  color="primary"
-                  disabled={!isValid || isSubmitting || creating}
-                >
-                  {creating && (
-                    <Spinner
-                      size="sm"
-                      color="white"
-                      data-testid="creating-spinner"
-                      className="ticket-create-spinner"
-                    />
-                  )}
-                  Add Ticket
-                </Button>
+                {isAuthenticated && (
+                  <Button
+                    type="submit"
+                    color="primary"
+                    disabled={!isValid || isSubmitting || creating}
+                  >
+                    {creating && (
+                      <Spinner
+                        size="sm"
+                        color="white"
+                        data-testid="creating-spinner"
+                        className="ticket-create-spinner"
+                      />
+                    )}
+                    Add Ticket
+                  </Button>
+                )}
+                {!isAuthenticated && (
+                  <Button
+                    type="submit"
+                    color="primary"
+                    disabled={disableSubmit}
```

Comment:
We should control `disabled` state with a combination of formik values and state of the ticket.

Reply:
I think these are formik values ` !isValid || isSubmitting || creating` ?